packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -77,8 +77,15 @@ class CollapsibleMenuView extends React.Component {
 
     for (let i = 0; i < this.props.children.length; i += 1) {
       const child = this.container.children[i];
-      calcWidth += child.getBoundingClientRect().width;
+      const childWidth = child.getBoundingClientRect().width;
+      calcWidth += childWidth;
+
       if (calcWidth > availableWidth) {
+        // If last child is the same size as the menu button, leave face up
+        if (i === this.props.children.length - 1 && childWidth === menuButtonWidth) {
```

Comment:
While this solves the specific scenario that was logged, it doesn't really solve the entire issue. This would still be broken in the scenario where a child button is larger than the menu button, but could stil fit in the available space if the menu was hidden. The logic should be something like:
`if (i === this.props.children.length - 1 && calcWidth <= availableWidth + menuButtonWidth)`